Session/Home/HomeViewModel.swift

```diff
@@ -578,16 +617,44 @@ public class HomeViewModel: NavigatableStateHolder {
         dependencies[defaults: .standard, key: .rateAppRetryAttemptCount] = 0
         
         if let scene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
-            SKStoreReviewController.requestReview(in: scene)
+            NotificationCenter.default.addObserver(
+                self,
+                selector: #selector(self.systemHasPresentedNewWindow(notification:)),
+                name: UIWindow.didBecomeVisibleNotification, object: nil
+            )
+            
+            if !dependencies[feature: .simulateAppReviewLimit] {
+                SKStoreReviewController.requestReview(in: scene)
```

Comment:
So it seems that `SKStoreReviewController` is deprecated (https://developer.apple.com/documentation/storekit/skstorereviewcontroller) - can you please update this to use `RequestReviewAction` instead? (https://developer.apple.com/documentation/storekit/requestreviewaction)